packages/client/hmi-client/src/components/workflow/util.ts

```diff
@@ -120,3 +120,53 @@ export const getGraphDataFromDatasetCSV = (
 export function getActiveOutput<S>(node: WorkflowNode<S>) {
 	return node.outputs.find((output) => output.id === node.active);
 }
+
+/**
+ * Get the documentation URL for a given node operation
+ * @param node - The node to get the documentation URL for
+ */
+export function getDocumentationUrl(node: WorkflowNode<any>): string {
+	switch (node.operationType) {
+		case WorkflowOperationTypes.CALIBRATION_CIEMSS:
+			return 'https://documentation.terarium.ai/simulation/calibrate-model/';
+		case WorkflowOperationTypes.CALIBRATE_ENSEMBLE_CIEMSS:
+			return 'https://documentation.terarium.ai/simulation/calibrate-ensemble/';
+		case WorkflowOperationTypes.COMPARE_DATASETS:
+			return 'https://documentation.terarium.ai/datasets/compare-datasets/';
+		case WorkflowOperationTypes.DATASET:
+			return 'https://documentation.terarium.ai/datasets/review-and-enrich-dataset/';
+		case WorkflowOperationTypes.DATASET_TRANSFORMER:
+			return 'https://documentation.terarium.ai/datasets/transform-dataset/';
+		case WorkflowOperationTypes.FUNMAN:
+			return 'https://documentation.terarium.ai/config-and-intervention/validate-model-configuration/';
+		case WorkflowOperationTypes.INTERVENTION_POLICY:
+			return 'https://documentation.terarium.ai/config-and-intervention/create-intervention-policy/';
+		case WorkflowOperationTypes.MODEL:
+			return 'https://documentation.terarium.ai/modeling/review-and-enrich-model/';
+		case WorkflowOperationTypes.MODEL_COMPARISON:
+			return 'https://documentation.terarium.ai/modeling/compare-models/';
+		case WorkflowOperationTypes.MODEL_CONFIG:
+			return 'https://documentation.terarium.ai/config-and-intervention/configure-model/';
+		case WorkflowOperationTypes.MODEL_EDIT:
+			return 'https://documentation.terarium.ai/modeling/edit-model/';
+		case WorkflowOperationTypes.MODEL_FROM_EQUATIONS:
+			return 'https://documentation.terarium.ai/modeling/create-model-from-equations/';
+		case WorkflowOperationTypes.OPTIMIZE_CIEMSS:
+			return 'https://documentation.terarium.ai/config-and-intervention/optimize-intervention-policy/';
+		case WorkflowOperationTypes.REGRIDDING:
+			return 'https://darpa-askem.github.io/askem-beaker/contexts_climate_data_utility.html';
+		case WorkflowOperationTypes.SIMULATE_CIEMSS:
+			return 'https://documentation.terarium.ai/simulation/simulate-model/';
+		case WorkflowOperationTypes.SIMULATE_ENSEMBLE_CIEMSS:
+			return 'https://documentation.terarium.ai/simulation/simulate-ensemble/';
+		case WorkflowOperationTypes.STRATIFY_MIRA:
+			return 'https://documentation.terarium.ai/modeling/stratify-model/';
+		case WorkflowOperationTypes.SUBSET_DATA:
```

Comment:
I think it is better if the documentation urls remain with the operators, such that it is self contained. I think it will be easier to maintain as the information is in one place/directory and not spilled over to some other component.

Something like
```
getDocumentationUrl(node) {
   return regstry.getOperator(node.operatorType).documentationURL
}
```

Reply:
The reason I did not do that, was because:
* that information `documentationURL` is only being used by the method `getDocumentUrl()` and being called only in the node header.
* I thought it will be heavy to load the full registry within the `utils.ts`
* I did not thought it would be a good place to put that funtion within `tera-workflow.vue`
* Another solution was to put the registry in a composables for ease of access... but that's an overkill just to display urls

But I will just try to implement the `getDocumentationUrl()` within `tera-workflow.vue` and use some system to pass it on to the drilldown.